src/index.html

```diff
@@ -3,11 +3,311 @@
+            alt="phone2">
+          <h3 class="section__title">
+            Ear (2)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Perfecting the path from artist to ear. So that music sounds just as it was intended.
+          </p>
+          <p class="section__price">
+            $ 149.00
+          </p>
+        </article>
+
+        <article class="section__article section__article--main">
+          <img
+            class="section__picture"
+            src="images/photos/section3.jpg"
+            alt="phone2">
+          <h3 class="section__title">
+            Ear (stick)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Tech you can &#x2019; feel. Leaving just space for sound to come to life. Powered by custom sound technology.
+          </p>
+          <p class="section__price">
+            $ 119.00
+          </p>
+        </article>
+      </section>
+
+      <h2 class="title title__secondary title__secondary--browse main__title" id="store">
+        Browse Nothing products by category
+      </h2>
+
+      <section class="section main__section">
+        <article class="section__article">
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/photos/all-products1.jpg"
+              alt="birds on phone"
+            >
+
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/photos/all-products2.jpg"
+              alt="headphones-box-1"
+            >
+            <h3 class="section__title section__title--second">
+              All products
+            </h3>
+          </div>
+        </article>
+
+        <article class="section__article">
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/photos/audio1.jpg"
+              alt="headphones"
+            >
+
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/photos/audio2.jpg"
+              alt="headphones-box-1"
+            >
+
+            <h3 class="section__title section__title--second">
+              Audio
+            </h3>
+          </div>
+        </article>
+
+        <article class="section__article">
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/photos/accessories1.jpg"
+              alt="mobilephone"
+            >
+
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/photos/accessories2.jpg"
+              alt="phone-charger"
+            >
+
+            <h3 class="section__title section__title--second section__title--button">
+              Accessories
+            </h3>
+          </div>
+
+          <a href="#" class="button">View all</a>
```

Comment:
Add proper hover effect for this button and add cursor pointer for form button. You can style those common buttons with the same class